src/analysis/convex/hull.lean

```diff
@@ -95,21 +109,6 @@ begin
     by { rintro (rfl : y = x), exact hx hy }⟩),
 end
 
-lemma is_linear_map.image_convex_hull {f : E → F} (hf : is_linear_map 𝕜 f) :
```

Comment:
We should still keep this one and the below (probably proved using the affine versions) for the sake of library search and ease of API use